packages/bpk-component-map/src/BpkMap-test.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import BpkMap from './BpkMap';
+
+const standardProps = {
+  googleMapURL: `https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places`,
+  loadingElement: <div />,
+  containerElement: <div />,
+  mapElement: <div />,
+};
+
+describe('BpkMap', () => {
+  let wrapper;
+  beforeEach(done => {
+    wrapper = mount(
+      <BpkMap
+        {...standardProps}
+        mapRef={() => {
+          done();
+        }}
+      />,
+    );
+  });
+
+  it('should render without throwing', () => {
+    expect(wrapper.exists()).toBeTruthy();
+  });
+});
```

Comment:
I'm struggling to provide value with this test. As we're just delegating all functionality to `react-google-maps`, there isn't much we can test.

Reply:
So don't test it? :)

In all seriousness we should only be testing our code and from the looks of it `BpkMap.js` doesn't have much of our code in it.

Reply:
No complaints from me!